recipes/nxviz/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
   md5: {{ md5 }}
 
 build:
-  number: 0
+  number: 0 
```

Comment:
Instead of adding a blank space you use `git ci --amend`, change the commit message, and force push.

Also, there is no re-rendering in staged-recipes :wink: only in the feedstock. Although, rebasing against master is recommended to get latest CI configuration.